src/components/MapContainer.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Map from './Map'
+import { connect } from 'react-redux'
+
+class MapContainer extends React.Component {
+  render() {
+    return (
+      <Map config={this.props.config} />
+    )
+}
```

Comment:
Whoops, it looks like the [preview build failed](https://app.netlify.com/sites/opentraffic-ui/deploys/595f86e57960b17e3720ccbc). Missing closing bracket?